sigs.yaml

```diff
@@ -1062,12 +1062,15 @@ sigs:
       description: Test Failures and Triage
   subprojects:
   - name: community
+    description: Owns and manages overall community repo, including SIG description and charter maintenance.
```

Comment:
I would note this as 'owns and manages overall community repo, including community group documentation and operations.' I think SIGs own their own charter maintenance and descriptions at the end of the day.